unit/src/MooseServerTest.C

```diff
@@ -1066,6 +1066,7 @@ TEST_F(MooseServerTest, CompletionDocumentRootLevel)
 label: active                           text: active = '__all__'                         desc: If specified only the blocks named wi... pos: [42.0]-[42.0] kind:  7
 label: inactive                         text: inactive = 'value'                         desc: If specified blocks matching these id... pos: [42.0]-[42.0] kind:  7
 label: Adaptivity                       text: [Adaptivity]\n  \n[]                       desc: application named block                  pos: [42.0]-[42.0] kind: 22
+label: Application                      text: [Application]\n  \n[]                      desc: application named block                  pos: [42.0]-[42.0] kind: 22
```

Comment:
I fixed this test you no longer have to change it. This line can stay, the other one will conflict